File: LeetCode/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False
        elif len(str(x)) == 1:
            return True
        check = [int(i) for i in str(x)]
        inverse = check[::-1]
        for i,value in enumerate(check):
            if check[i] == inverse[i]:
                pass
            else:
                return False
        return True

if __name__ == "__main__":
    solution  = Solution()
    number = 999
    result = solution.isPalindrome(number)
    print(result)

File: Leetcode Javascript/palindrome-number.js
var isPalindrome = function(x) {
    if ( x < 0) {
        return false
    } else if (x.toString().length == 1) {
        return true
    }
    let my_array = Array.from(String(x), Number)
    let inverse = [...my_array].reverse()
    for (let [i, n] of my_array.entries()) {
        if (my_array[i] === inverse[i]) {
        } else {
            return false
        }
    }
    return true
};

console.log(isPalindrome(15))